issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraIssue.java

```diff
@@ -261,6 +265,7 @@ public void setState(State state) {
                     }
                 } else {
                     // The issue is most likely closed - skip transitioning
+                    log.warning("Can't transition the issue to Resolved or Open");
```

Comment:
@erikj79 You suggested that 

> The else clause should at the very least log something so we know that is what happened. we should also verify that the state is either closed or resolved and throw RuntimeException if not. 

I thought about it, then I realized that the Skara bot should have already verified the state before calling Issue.setState(Issue.State.RESOLVED) like [this](https://github.com/openjdk/skara/blob/b8a99e211aae1cec3ae5b6477e9c0c91c0761284/bots/notify/src/main/java/org/openjdk/skara/bots/notify/issue/IssueNotifier.java#L211).

Also, it's will be a little hard to verify the Issue state since we need to change the interface of Issue.setState.

Reply:
I think logging is good enough for now.